Assets/Scripts/Rendering/State.cs

```diff
@@ -1,17 +1,25 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 
 
 public class State : MonoBehaviour
 {
     // TODO: Change, right now dynamic updates do not work with Queue areas
-    private const float UPDATE_INTERVAL = 100000.0f;
+    private const float UPDATE_INTERVAL = 10.0f;
     private float updateCountdown = UPDATE_INTERVAL;
 
     // Main dictionary keeping all connection and their rendered counterparts
     private Dictionary<MQ.Client, GameObject> qmgrs = new Dictionary<MQ.Client, GameObject>();
 
+    // Counter for recording the number of rendered QM
+    private int renderedQueueManagers = 0;
+    // Integer array for recording the postion of the last rendered manager
+    private List<int[]> lastQMPosition = new List<int[]>();
```

Comment:
This list seems to contain all the rendered QM's positions, so maybe it should be called QM Positions?